server.js

```diff
@@ -1,13 +1,56 @@
 const express = require('express');
+const redis = require('redis');
+const bodyParser = require('body-parser');
+
 
 const app = express();
+const redisClient = redis.createClient();
+
+app.use(bodyParser.json());
+
+app.get('/:key', (req, res) => {
+
+    const key = req.params.key
 
-app.get('/:id', (req, res) => {
+    redisClient.get(key, (err, reply) => {
 
```

Comment:
Use Promises/Async Await instead of callbacks. It’s a good node.js programming practice and plus you guys should know how it works.